src/v2/api/controllers/AuthController.ts

```diff
@@ -143,6 +143,18 @@ export class AuthController {
     return this.authService.refresh(req.user);
   }
 
+  @ApiBearerAuth()
+  @ApiOkResponse({
+    type: AuthLoginResponse,
+  })
+  @ApiBadRequestResponse({
+    description: `\n
+    InvalidEntityIdException:
+      Entity with such id is not found
+
+    PasswordRepeatException:
+      The passwords are the same`,
+  })
   @UseGuards(JwtGuard)
   @Put('/updatePassword')
   async updatePassword (
```

Comment:
You need to describe the body of the request to this endpoint, namely UpdatePasswordDTO

Reply:
How do I do this? In UpdatePasswordDTO.ts?